components/ActionSheet/tests/unit/MDCActionSheetThemeTest.m

```diff
@@ -98,7 +107,7 @@ - (void)testApplyThemerWithOnlyMessage {
 
 #pragma mark - default test
 
-- (void)testApplyThemeToCells {
+- (void)testApplyColorThemeToCells {
```

Comment:
Suggestion: Consider `testApplyColorTheme` to be consistent with the method name below.